backend/editor/controllers/project_controller.py

```diff
@@ -13,7 +15,10 @@ async def get_project(project_id: str) -> Project:
     """
     params = {"project_id": project_id}
     result = await get_current_transaction().run(query, params)
-    return Project(**(await result.single())["p"])
+    project = (await result.single())["p"]
+    if project is None:
+        raise HTTPException(status_code=404, detail="Project not found")
+    return Project(**project)
```

Comment:
@eric-nguyen-cs If the `project_id` does not exist in the database, this will cause a `TypeError: 'NoneType' object is not subscriptable` error because we try to access a property of a `None` object.

```suggestion
    project = await result.single()
    if project is None:
        raise HTTPException(status_code=404, detail="Project not found")
    return Project(**project["p"])
```

Reply:
Thanks!